code/game/vehicles/spacepods/equipment.dm

```diff
@@ -186,3 +187,42 @@
 /obj/item/device/spacepod_equipment/cargo/ore/removed(var/mob/user)
 	. = ..()
 	unload()
+
+/obj/item/device/spacepod_equipment/lock
+	name = "pod lock"
+	desc = "You shouldn't be seeing this"
+	icon = 'icons/goonstation/pods/ship.dmi'
```

Comment:
Well I have new sprites for this from flattest, so I can start implementing those over the various systems including this one. I just need to add them to a new icon file and they'll be good to go.